xwiki-pro-macros-test/xwiki-pro-macros-test-docker/src/test/it/com/xwiki/pro/test/ui/GenericMacrosIT.java

```diff
@@ -76,38 +90,170 @@
     })
 public class GenericMacrosIT
 {
+    private static final List<String> BASE_XWIKI_MACRO_SPACE = List.of("XWiki", "Macros");
+
     private final DocumentReference pageWithTeamMacros = new DocumentReference("xwiki", "Main", "TeamTest");
 
-    private static final String PAGE_WITH_TEAM_MACROS_CONTENT = "{{team/}}\n"
+    private final DocumentReference pageWithButtonMacros = new DocumentReference("xwiki", "Main", "ButtonTest");
+
+    private final DocumentReference pageWithStatusMacros = new DocumentReference("xwiki", "Main", "StatusTest");
+
+    private final DocumentReference pageWithTagListMacros = new DocumentReference("xwiki", "Main", "TagListTest");
+
+    private final DocumentReference pageWithTags = new DocumentReference("xwiki", "Main", "pageWithTags");
+
+    private final DocumentReference pageWithTags2 = new DocumentReference("xwiki", "XWiki", "pageWithTags2");
+
+    private final DocumentReference pageWithExpandMacro = new DocumentReference("xwiki", "Main", "ExpandTest");
+
+    private final DocumentReference pageWithUserProfileMacro = new DocumentReference("xwiki", "Main",
+        "UserProfileTest");
+
+    private final DocumentReference pageWithUserListMacro = new DocumentReference("xwiki", "Main",
+        "UserListTest");
+
+    private final DocumentReference PageWithPanelMacros = new DocumentReference("xwiki", "Main",
+        "PanelTest");
+
+    private final DocumentReference PageWithMStreamMacros = new DocumentReference("xwiki", "Main",
+        "MicrosoftStreamTest");
+
+    private static final String PAGE_WITH_TEAM_MACROS_CONTENT = "{{team letterAvatarFontColor=\"rgb(204, 0, 255)\"/}}\n"
         + "\n"
-        + "{{team tag=\"testTag\" /}}\n"
+        + "{{team tag=\"testTag\" showUsernames=\"true\" disableTools=\"true\" disableLetterAvatars=\"true\" requireExternalAuth=\"false\" /}}\n"
         + "\n"
-        + "{{team tag=\"nonExistentTag\" /}}";
+        + "{{team tag=\"nonExistentTag\"  /}}";
 
+    private static final String PAGE_WITH_BUTTON_MACROS_CONTENT =
```

Comment:
Since you have these many macros, and some have a lot of content, I would suggest to move these in proper `.vm` files, which will be easier to be maintained 

You could use [test resources](https://dev.xwiki.org/xwiki/bin/view/Community/Testing/DockerTesting/#HTestResources) and create for each macro a new vm file with its content. After, you would use these to access the content in your test (e.g. https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-docker/src/test/it/org/xwiki/flamingo/test/docker/AttachmentIT.java#L201 )